### ISSUE TITLE
v5 error on config reload - web.config

### ISSUE THREAD

**Issue by AndersMad** [NLog/NLog#4575]
Error when reloading the config. Worked on latest release / non-review. Config is in web.config.

**NLog version**: v5.preview.1

**Error:** `Unrecognized value 'xmlns'='http://www.nlog-project.org/schemas/NLog.xsd' for element 'nlog'`

**Code:** NLog.LogManager.Configuration.Reload()

**Platform**: .Net 4.8

**Current NLog config:**
```xml
<configuration>
	<configSections>
		<section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog"/>		
	</configSections>
	<nlog throwExceptions="true" xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"> ...
```

**Comment by snakefoot** [comment]
Thank you for reporting this. The bug has been fixed with #4533 + #4555

Hopefully there will soon be a NLog 5.0-preview2 release, but there is some pending work on resolving #2249

Notice that it is NOT recommended to use `throwExceptions="true"` in production-code, since the setting is for unit-testing (and sometimes last resort troubleshooting).

**Comment by snakefoot** [comment]
Released ver. 5.0-preview-2 https://www.nuget.org/packages/NLog/5.0.0-preview.2
